tests/test_diffraction_fft.py

```diff
@@ -0,0 +1,477 @@
+        image = galsim.config.BuildImage(config, logger=logger)
+    # To save the image produced here, use:
+    # galsim.fits.write(image, file_name="/tmp/spikes.fits")
+    brightness = image.array
+    # Center of star:
+    [c_x, c_y] = center_of_brighness(brightness)
+    # 2 Pixel tolerance:
+    np.testing.assert_allclose(
+        np.array([c_x, c_y]), np.array([1923.0, 3161.0]), atol=2.0, rtol=0.0
+    )
+    angle, angle_stddev = folded_spike_angle(brightness, c_x, c_y, r_min=10.0)
+    expected_angle = (
+        45.0 * galsim.degrees - config["stamp"]["diffraction_psf"]["rottelpos"]
+    ).deg
+    # 1° tolerance:
+    np.testing.assert_allclose(np.rad2deg(angle), expected_angle, atol=1.0, rtol=0.0)
+    expected_max_spike_width = 5.0
+    np.testing.assert_array_less(np.rad2deg(angle_stddev), expected_max_spike_width)
+    slope, intercept, slope_stderr, intercept_stderr = brightness_log_profile(
+        brightness, c_x, c_y
+    )
+    np.testing.assert_allclose(slope, -2.0, atol=0.6, rtol=0.0)
+    np.testing.assert_allclose(intercept, 12.5, atol=3.0, rtol=0.0)
+    # Assert some rough evidence, that brightness decays as
+    # brightness ~ exp(intercept) * r**slope:
+    np.testing.assert_array_less(slope_stderr, 0.2)
+    np.testing.assert_array_less(intercept_stderr, 0.8)
+
+
+def test_fft_diffraction_is_similar_to_raytracing_for_field_rotation():
+    """Check, that raytracing diffraction and FFT diffraction are "similar"."""
+    config = create_test_config(XSIZE, YSIZE, STAMP_SIZE, exptime=300.0)
+    # To compare with the raytracing method, instead use:
+    # config = create_test_config(XSIZE, YSIZE, STAMP_SIZE, exptime=300.0, mode=Mode.RAYTRACING)
+
+    with assert_no_error_logs() as logger:
+        image = galsim.config.BuildImage(config, logger=logger)
+    # To save the image produced here, use:
+    # galsim.fits.write(image, file_name="/tmp/spikes.fits")
+    brightness = image.array
+    [c_x, c_y] = center_of_brighness(brightness)
+    # 2 Pixel tolerance:
+    np.testing.assert_allclose(
+        np.array([c_x, c_y]), np.array([1923.0, 3161.0]), atol=2.0, rtol=0.0
+    )
+    angle, angle_stddev = folded_spike_angle(brightness, c_x, c_y)
+    expected_spike_width = 14.4
+    expected_angle = (
+        45.0 * galsim.degrees - config["stamp"]["diffraction_psf"]["rottelpos"]
+         - expected_spike_width / 2.0 * galsim.degrees
+    ).deg
+    # 2.5° tolerance (we neglect the time dependence of the rotation rate):
+    np.testing.assert_allclose(np.rad2deg(angle), expected_angle, atol=2.5, rtol=0.0)
+    np.testing.assert_allclose(
+        np.rad2deg(angle_stddev), expected_spike_width / 2., atol=1.0, rtol=0.0
+    )
+    slope, intercept, slope_stderr, intercept_stderr = brightness_log_profile(
+        brightness, c_x, c_y
+    )
+    # Brightness should decay as ~1/r**2:
+    np.testing.assert_allclose(slope, -2.0, atol=0.6, rtol=0.0)
+    np.testing.assert_allclose(intercept, 12.6, atol=3.0, rtol=0.0)
```

Comment:
OK, here is an explanation for the slope=-2.  But I why 12.6?  And 12.5 above?  With tolerance of +- 3?

Reply:
Regarding the intercept value, I just was not sure on how close this should come to the raytracing value / dont have a good intuition for this value. Thus I just left it here as an additional regression test.
Also: any suggestion on how to deal with this value is appreciated.